app/services/address_book.rb

```diff
@@ -0,0 +1,13 @@
+class AddressBook
+  def initialize(domain)
+    @domain = domain
+  end
+
+  def feedback
+    "feedback@#{@domain}"
```

Comment:
`feedback` doesn't, because it's just a stand-in for sending things to Zendesk, and the public won't ever see it. `no-reply` is an interesting one, though!